virtualization/windowscontainers/deploy-containers/containerd.md

```diff
@@ -0,0 +1,110 @@
+
+![Future Docker Engine architecture on Windows](media/hcs-with-runhcs.png)
+
+## runhcs
+
+RunHCS is a fork of runc.  Like runc, runhcs is a command line client for running applications packaged according to the Open Container Initiative (OCI) format and is a compliant implementation of the Open Container Initiative specification.  The biggest difference between the two, besides running on Windows, is that runhcs can run both Windows and Linux [Hyper-V containers](../manage-containers/hyperv-container.md) in addition to Windows process containers.
+
+Usage:
+
+``` cmd
+runhcs run [ -b bundle ] <container-id>
+```
+
+`<container-id>` is your name for the container instance you are starting. The name must be unique on your container host.
+
+The bundle directory (using `-b bundle`) is optional.  
+As with runc, containers are configured using bundles. A container's bundle is the directory with the container's OCI specification file, "config.json".  The default value for "bundle" is the current directory.
+
+The OCI spec file, "config.json", has to have two fields to run correctly:
+
+1. A path to the container's scratch space
+1. A path to the container's layer directory
+
+Container commands available in runhcs include:
+
+* Tools to create and run a container
+  * **run** creates and runs a container
+  * **create** create a container
+  * **exec** runs a new process inside the container
+
+* Tools to manage processes running in a container:
+  * **start** executes the user defined process in a created container
+  * **pause** pause suspends all processes inside the container
+  * **resume** resumes all processes that have been previously paused
+  * **ps** ps displays the processes running inside a container
+
+* Tools to manage a container's state
+  * **state** outputs the state of a container
+  * **kill** sends the specified signal (default: SIGTERM) to the container's init process
+  * **delete** deletes any resources held by the container often used with detached container
+
+The only command that could be considered multi-container is **list**.  It lists running (or paused) containers started by runhcs with the given root.
+
+## containerd/cri
+
+> !NOTE CRI support is only available in Server 2019/Windows 10 1809 and later.
+
+While OCI specs define a single container, CRI (container runtime interface) spec defines a multi-container environment for container orchestrators like Kubernetes and Service Fabric Mesh.
+
+![Containerd based container environments](media/containerd-platform.png)
+
+While runHCS and containerd both can manage on any Windows system Server 2016 or later, supporting Pods (groups of containers) required breaking changes to container tools in Windows.  CRI support is available on Windows Server 2019/Windows 10 1809 and later.
```

Comment:
NIT: `runhcs` is the correct casing. All lower

Reply:
👍